Fix crash in function winMultiWindowGetTransientFor() (fd.o bugzilla #11147)

The function winMultiWindowGetTransientFor() crashes, due to a bogus pointer
dereference, but this fact was previously hidden by the function not being called!

---
 xserver/hw/xwin/winmultiwindowclass.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: xorg-server-1.5.3/xserver/hw/xwin/winmultiwindowclass.c
===================================================================
--- xorg-server-1.5.3.orig/xserver/hw/xwin/winmultiwindowclass.c	2009-02-16 14:24:08.734375000 +0000
+++ xorg-server-1.5.3/xserver/hw/xwin/winmultiwindowclass.c	2009-02-16 14:25:41.765625000 +0000
@@ -1,5 +1,6 @@
 /*
  *Copyright (C) 1994-2000 The XFree86 Project, Inc. All Rights Reserved.
+ *Copyright (C) Colin Harrison 2005-2008
  *
  *Permission is hereby granted, free of charge, to any person obtaining
  * a copy of this software and associated documentation files (the
@@ -26,6 +27,7 @@
  *from the XFree86 Project.
  *
  * Authors:     Earle F. Philhower, III
+ *              Colin Harrison
  */
 
 #ifdef HAVE_XWIN_CONFIG_H
@@ -263,7 +265,7 @@
       if (prop->propertyName == XA_WM_TRANSIENT_FOR)
         {
           if (ppDaddy)
-            memcpy (*ppDaddy, prop->data, sizeof (WindowPtr));
+            memcpy (ppDaddy, prop->data, sizeof (WindowPtr));
           return 1;
         }
       else
